app/controllers/collection_controller.rb

```diff
@@ -326,37 +326,41 @@ def contributors_download
     ]
 
     stats = @active_transcribers.map do |user|
-      time_total = 0
-      time_proportional = 0
+      if user.notification.user_activity?
```

Comment:
This doesn't work.  Project owners need to know how much work is happening.  The intent of this fix is just to keep project owners from emailing folks.  We should just skip printing their email address in this report, but include their username and time contributed.

Reply:
@saracarl OK, I changed the code to show emails as "opt out" for users who have their user_activity set to false. Please let me know if this works.